dart/lib/src/http_client/failed_request_client.dart

```diff
@@ -165,6 +167,7 @@ class FailedRequestClient extends BaseClient {
       url: urlWithoutQuery,
       queryString: query,
       cookies: sendDefaultPii ? request.headers['Cookie'] : null,
+      // TODO FIXME - this should check [sendDefaultPii], same as DIO integration.
       data: _getDataFromRequest(request),
```

Comment:
This is an open issue. I'd say body is PII and the DIO integration treats it the same. It's not in the scope of this PR per-se, just something I've noticed here.